cmd/noms/noms.go

```diff
@@ -68,12 +67,12 @@ func main() {
 	for _, cmd := range commands {
 		if cmd.Name() == args[0] {
 			flags := cmd.Flags()
-			version.RegisterVersionFlags(*flags)
```

Comment:
The command line utilities and programs were reverted back to how they were before my prior change.